perfkitbenchmarker/providers/gcp/gce_network.py

```diff
@@ -54,6 +55,7 @@ def _Create(self):
                   'firewall-rules',
                   'create',
                   self.name,
+                  '--network=%s' % self.network.network_resource.name,
                   '--allow', 'tcp:%d' % self.port, 'udp:%d' % self.port]
     create_cmd.extend(util.GetDefaultGcloudFlags(self))
```

Comment:
Because of a change a merged, the old process of build gcloud command -> call util.GetDefaultGcloudFlags -> call vm_util.IssueCommand is replaced by creating a GcloudCommand object and calling its Issue/IssueRetryable method. Sorry about the hassle.